jujugui/static/gui/src/app/components/inspector-relations/test-inspector-relations.js

```diff
@@ -32,7 +32,8 @@ describe('InspectorRelations', function() {
   });
 
   it('can render the onboarding', function() {
-    var getRelationDataForService = sinon.stub.returns([]);
+    var getRelationDataForService = sinon.stub();
+    getRelationDataForService.returns([]);
```

Comment:
The 'old' version was actually returning a stub function instead of the array. This fixed that, maybe they can't be chained like that?